bigquery/snippets/job/bigquery_extract_table_json.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package job
+
+// [START bigquery_extract_table_json]
+
+import (
+	"cloud.google.com/go/bigquery"
+	"context"
+	"fmt"
+)
+
+func exportSampleTableAsJSON(projectID, gcsURI string) error {
```

Comment:
Delete `Sample`?

Reply:
s/Sample//g done here and in loadingdata